articles/search/search-howto-index-json-blobs.md

```diff
@@ -44,7 +43,7 @@ Alternatively, when your blobs contain an **array of JSON objects**, you may wan
 you can populate your Azure Search index with three separate documents, each with "id" and "text" fields.
 
 > [!IMPORTANT]
-> The JSON array parsing functionality is currently in preview. It is available only in the REST API using version **2015-02-28-Preview**. Remember, preview APIs are intended for testing and evaluation, and should not be used in production environments.
+> The JSON array parsing functionality is currently in preview. It is available only in the REST API using version **2016-09-01-Preview**. Remember, preview APIs are intended for testing and evaluation, and should not be used in production environments.
```

Comment:
We haven't done a sweep of all our docs to update preview API version numbers. If you see any more like this, feel free to update them as well.